tutugodfrey/Pro-tutor
# #161571582 set up test environment for the project

#### What does this PR do?
- Set up tests for the project

#### Description of Task to be completed?
- Install packages required for testing
- integrate code-climate for test coverage reporting
- Integrate coveralls for tests coverage reporting
- Add coverage badges to the README

#### How should this be manually tested?
- clone the repo and check out the branch. Run `npm install` to install dependencies. Then run `npm test` to run the test that is present. Confirm that the test passes.

#### What are the relevant pivotal tracker stories?
#161571582
